build/xtask/src/dist.rs

```diff
@@ -549,6 +555,70 @@ pub fn package(
     Ok(allocated)
 }
 
+// can we use lpc55-romapi
+const LPC55_FLASH_PAGE_SIZE: usize = 512;
```

Comment:
Hmmmm, I think what we actually want is like `lpc55-rom-constants` or something similar (currently running into this with some of the CMPA/CFPA reading as well). Probably not worth blocking for it.

Reply:
I've added a new commit addressing this feedback. Only the constant used here by `xtask` was moved to the new `lpc55-rom-data` crate. Others can migrate on demand. I'll squash all of this down once the PR is ready. LMK if this wasn't what you were after.